mmv1/third_party/terraform/services/bigtable/resource_bigtable_gc_policy.go

```diff
@@ -302,7 +336,7 @@ func resourceBigtableGCPolicyRead(d *schema.ResourceData, meta interface{}) erro
 		// Only set `gc_rules`` when the legacy fields are not set. We are not planning to support legacy fields.
 		maxAge := d.Get("max_age")
 		maxVersion := d.Get("max_version")
-		if d.Get("mode") == "" && len(maxAge.([]interface{})) == 0 && len(maxVersion.([]interface{})) == 0 {
```

Comment:
I'm guessing this is related to the "unpopulated gc_rules" part of the PR - could you elaborate on what the issue is that's being resolved? Is it covered by the tests added in this PR?

Reply:
We noticed that when a user indicated mode, but does not indicate max age or version, they will get infinite loop of update, as the update rules won't ever be populated in local state.
This is partly covered in the acc tests